instrumentation/aws-sdk/aws-sdk-1.11/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/awssdk/v1_11/TracingRequestHandler.java

```diff
@@ -49,6 +85,32 @@ public void afterResponse(Request<?> request, Response<?> response) {
     tracer().end(scope.getContext(), response);
   }
 
+  private boolean isSqsConsumer(AmazonWebServiceRequest request) {
+    return request.getClass().getName().endsWith("ReceiveMessageRequest");
+  }
+
+  /** Create and close CONSUMER span for each message consumed. */
```

Comment:
Yeah it's unclear whether in modeling we're supposed to even have these 0-timed spans, but the spec is vague on this so I'm ok with anything.

Reply:
OK :)